fetch.bs

```diff
@@ -5225,13 +5330,17 @@ these steps:
   <a>cross-origin resource policy check</a> with <var>request</var>'s <a for=request>origin</a>,
   <var>request</var>'s <a for=request>client</a>, <var>request</var>'s
   <a for=request>destination</a>, and <var>internalResponse</var> returns <b>blocked</b>, then
-  return a <a>network error</a>.
+  return the result of running <a>HTTPS upgrade fallback</a> given <var>request</var>
```

Comment:
Should we fallback on this CORS error?

Reply:
Probably not, as this shouldn't apply to subresources. Removed.